src/renderer/components/query.jsx

```diff
@@ -157,6 +157,34 @@ export default class Query extends Component {
     });
   }
 
+  getCommands () {
```

Comment:
Could have a separate menu for that. Which would not cause this misunderstanding. But I guess is better keep it simple for now.